src/pages/UserHome.jsx

```diff
@@ -110,34 +102,43 @@ export default function UserHome() {
           </Button>
         </Grid>
       </Grid>
-      <Box className={classes.noScholarshipsBox}>
-        <InboxIcon style={{ fontSize: 200 }} />
-        <Typography
-          variant="h5"
-          component="h2"
-          gutterBottom
-          className={classes.noScholarshipsAdded}>
-          No Scholarships Added Yet
-        </Typography>
-
-        <Typography component="h2" color="primary">
-          <MuiLink component={Link} to="/scholarship/new">
-            Add a Scholarship
-          </MuiLink>
-        </Typography>
-      </Box>
       {error?.toString() ||
         (loading && <CircularProgress className={classes.progress} />) || (
           <>
+            {scholarships.length === 0 && (
+              <Grid
+                container
+                justify="flex-start"
+                alignItems="center"
+                className={classes.bannerRoot}>
+                <Grid item>
+                  <InboxIcon style={{ fontSize: 200 }} />
+                </Grid>
+                <Grid item>
+                  <Typography variant="h5" gutterButtom>
+                    No Scholarships Added Yet
+                  </Typography>
+                  <Typography
+                    variant="contained"
+                    color="primary"
+                    component={Link}
+                    to="/scholarships/new">
+                    Add Scholarship
+                  </Typography>
+                </Grid>
+              </Grid>
+            )}
             <ScholarshipList scholarships={scholarships} />
-            <Button
-              className={classes.loadMoreButton}
-              color="primary"
-              onclick={() => {
-                alert('clicked');
-              }}>
-              Load More
-            </Button>
+            {scholarships.length !== 0 && (
+              <Button
+                className={classes.loadMoreButton}
+                color="primary"
+                onclick={() => {
+                  alert('clicked');
+                }}>
+                Load More
+              </Button>
+            )}
```

Comment:
I'm not sure how you've read into [conditional rendering](https://reactjs.org/docs/conditional-rendering.html) but there's a section on [inlining if-else](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator) within the JSX. We generally use that where possible. But, since this needs to be updated anyways when the **Load More** button works, making this change is optional.